cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -207,8 +207,10 @@ function deploy_sriov_operator {
 
   echo 'Generating webhook certificates for the SR-IOV operator webhooks'
   pushd "${CERTCREATOR_PATH}"
-    go run . -namespace sriov-network-operator -secret operator-webhook-service -hook operator-webhook -kubeconfig $KUBECONFIG_PATH || return 1
-    go run . -namespace sriov-network-operator -secret network-resources-injector-secret -hook network-resources-injector -kubeconfig $KUBECONFIG_PATH || return 1
+    go build -o certcreator .  
+    ./certcreator -namespace sriov-network-operator -secret operator-webhook-service -hook operator-webhook -kubeconfig $KUBECONFIG_PATH || return 1
+    ./certcreator -namespace sriov-network-operator -secret network-resources-injector-secret -hook network-resources-injector -kubeconfig $KUBECONFIG_PATH || return 1
+    rm -f certcreator
```

Comment:
if the previous lines failed, you wont delete it,
also please make sure its in .gitignore

Reply:
done

Reply:
i am little confused, you fixed it but totally removed `wait.Poll`
which will make this solution flake as we saw `etcd timeouts`
https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/472/check-up-kind-1.17-sriov/1326070455454404610

Once you are ready for the next cycle please add a message about the changes in the root of the PR

Reply:
still doesnt delete certcreator when the lines `return 1`
its better now that it ignores it, but i would suggest to add it to a trap please

Reply:
Sorry I was working with the wrong commits
This script is going to run inside a container so once finished its all gone, and in case it runs on a development environment once could debug what went wrong with the binary and exterminate it

Reply:
about the dev env, one can just compile it if needed, sources didnt change imo

Reply:
I will revert it to use `go run`